project-base/storefront/prettier.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-    plugins: [require('prettier-plugin-tailwindcss')],
+    plugins: [require('prettier-plugin-tailwindcss'), '@trivago/prettier-plugin-sort-imports'],
```

Comment:
I just know we had this plugin and decided to remove it (I do not even know why or who wanted it). Can we just make sure we want this step? If it had already been made sure, you can resolve this comment.

Reply:
According to chat conversation between FE guys we want to keep this.